site/AntBlazor.Docs/Demos/Components/Tooltip/demo/Basic.razor

```diff
@@ -1,3 +1,9 @@
 ﻿<Tooltip Title="@("prompt text")">
-    <span>Tooltip will show on mouse enter.</span>
+    <span>Tooltip (bound to div) will show on mouse enter.</span>
+</Tooltip>
+<br />
+<Tooltip Title="@("prompt text")">
+    <Unbound>
+        <span @ref="@context.Current">Tooltip (unbound) will show on mouse enter.</span>
+    </Unbound>
 </Tooltip>
```

Comment:
I feel that `Unbound` is not easy for users to understand. Is there a more direct and precise word for it?

Reply:
Generally I do no like it either, but we have to somehow communicate this is different approach. The ```Unbound``` is widely used in VB and this is where I took my inspiration from, but it was just a quick hack. Since the idea is to keep both solutions, somehow we need to communicate to the consumers which solution is being currently used. We are already here at a disadvantage, because we won't communicate (I guess at least) that the consumer is currently using div version of a component (so without reading docs, this won't be clear - I image there will be a lot of questions about it in the future). Best way would be maybe for the consumers to express what would be most intuitive? Otherwise it's just us giving proposals that may be clear to us but not necessarily to others. 
However, I can also give maybe a few proposals:
- ```<divless>```
- ```<wrapperless>```
- ```<wrapless>```
- ```<boxless>```
- ```<unboxed>```  

These are couple of suggestions. I was trying to suggest something that may directly indicate no div. Mind that English is not my primary language, so there's that...

Reply:
Users would wonder why there is a bound 😂

Reply:
Well...we could always change ```ChildContent``` to ```Bound``` or something similar just to indicate that there are 2 options, but that idea will create a breaking change and probably will make users scratch their heads on why there is even such option like div-bounded.

So. I see 3 options:
1. Keep 2 solutions - 1 with ```ChildContent``` (as is) and 1 with whatever we are going to name it (the currently used ```Unbound```)
2. Introduce a breaking change - and go with (for the sake of argument) ```Bound``` and ```Unbound``` versions
3. Introduce a breaking change - get rid of div-bounded solution and make current ```Unbound``` a default (thus removing the necessity to use ```<Unbound>``` tag at all) but force the users to use ```@context``` in their child element (that ```@context``` is the breaking change). 

The question you probably should ask yourself is - eventually, which solution should be preferred? As a full disclosure - I am obviously biased and I think it is clear I will try to promote the solution (or at least the idea) I presented here. However I feel that eventually the ```Unbound``` version should become the default, which in turn will introduce a breaking change. Since ant-design-blazor is still at its 0.x version, it is expected for it to keep evolving in violate ways which would create breaking changes. If that is going to be the case, then maybe it would be good to keep the ```Unbound``` version as is without spending too much time on figuring out a name that anyway eventually will be retired. We could keep the ```Unbound``` as is for now, see how it is going to work, fix any issues that are going to arise. And once you decide to make it default, then introduce a breaking change.

Reply:
Thank you, I'm prefer to the first optio :)

Reply:
So the ```<Unbound>``` tag stays? Are there any other suggestions? Shall I start working on other components that rely on ```OverlayTrigger```?

Reply:
Keep `<Unbound>`, and introduce the difference between with the `ChildContent`. 
Yeah, you can please continue to work on the other components.

Reply:
Please check commit 8eb8a04. I have no clue on how to approach documentation changes especially when it deviates in functionality compared to react ant-design. 
This is only in English...

Reply:
It's fine, and I will translate it to Chinese doc.